net/bind/Makefile

```diff
@@ -21,7 +21,7 @@ PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:= \
 	https://www.mirrorservice.org/sites/ftp.isc.org/isc/bind9/$(PKG_VERSION) \
```

Comment:
Replace with officially listed mirrors (https://ftp.isc.org/isc/MIRRORS)
https://ftp.funet.fi/pub/mirrors/ftp.isc.org/isc/bind9/
http://ftp.iij.ad.jp/pub/network/isc/bind9/
http://isc.mirrors.pair.com/bind9/

Reply:
That's a good idea, but we've been using the current mirror list for long enough that I wouldn't reject this PR for it.